dev/io.openliberty.security.jakartasec.3.0.internal/resources/io/openliberty/security/jakartasec/internal/resources/JakartaSecurity30Messages.nlsprops

```diff
@@ -0,0 +1,36 @@
+JAKARTASEC_WARNING_OIDC_MECH_CONFIG.explanation=The attribute could not be resolved as an expression. The EL result and the expected attribute value type could be mismatched. For example, if the expected attribute type is String, then the EL result must be a String.
+JAKARTASEC_WARNING_OIDC_MECH_CONFIG.useraction=Make sure that the annotation contains a valid configuration value. Ensure that the EL expressions are valid, that any referenced beans that are used in the expression are resolvable, and that the type of the result corresponds with the attribute.
+
+JAKARTASEC_WARNING_CLAIM_DEF_CONFIG=CWWKS2501W: The Expression Language (EL) expression for the {0} attribute of the claims definition annotation cannot be resolved to a valid value. The value provided was ''{1}''. Ensure that the EL expression and the result are valid and ensure that any referenced beans that are used in the expression are resolvable. The default attribute value of ''{2}'' is used.
+JAKARTASEC_WARNING_CLAIM_DEF_CONFIG.explanation=The attribute could not be resolved as an expression. The EL result and the expected attribute value type could be mismatched. For example, if the expected attribute type is String, then the EL result must be a String.
+JAKARTASEC_WARNING_CLAIM_DEF_CONFIG.useraction=Make sure that the annotation contains a valid configuration value. Ensure that the EL expressions are valid, that any referenced beans that are used in the expression are resolvable, and that the type of the result corresponds with the attribute.
+
+JAKARTASEC_WARNING_LOGOUT_DEF_CONFIG=CWWKS2502W: The Expression Language (EL) expression for the {0} attribute of the logout definition annotation cannot be resolved to a valid value. The value provided was ''{1}''. Ensure that the EL expression and the result are valid and ensure that any referenced beans that are used in the expression are resolvable. The default attribute value of ''{2}'' is used.
+JAKARTASEC_WARNING_LOGOUT_DEF_CONFIG.explanation=The attribute could not be resolved as an expression. The EL result and the expected attribute value type could be mismatched. For example, if the expected attribute type is String, then the EL result must be a String.
+JAKARTASEC_WARNING_LOGOUT_DEF_CONFIG.useraction=Make sure that the annotation contains a valid configuration value. Ensure that the EL expressions are valid, that any referenced beans that are used in the expression are resolvable, and that the type of the result corresponds with the attribute.
+
+JAKARTASEC_WARNING_PROV_METADATA_CONFIG=CWWKS2502W: The Expression Language (EL) expression for the {0} attribute of the OpenID provider metadat annotation cannot be resolved to a valid value. The value provided was ''{1}''. Ensure that the EL expression and the result are valid and ensure that any referenced beans that are used in the expression are resolvable. The default attribute value of ''{2}'' is used.
```

Comment:
There is a small typo in the sentence. `metadat` should be `metadata`.

Thanks!

Reply:
Good eye! Thanks.